### PR TITLE
Revised readme.md

### PR DESCRIPTION
**Pull request by uurcakmak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
-usage :
+## Under The Hood
 
-``` 
-py schema_exporter.py -s SCHAME_NAME -p PASSWORD -t TNS_NAME
+This script creates functions named `fetch_schema_metadata` and `gen_insert` in the Oracle Instance. The function `fetch_schema_metadata` fed through `all_objects` view and the result saved to disk as a single sql file per object.
+
+## Prerequisites
+
+- Python 3
```
Comment (erdemsarigh) [inline README.md:9]: yorumlarım 1

File: README.md [hunk 2]
```diff
@@ -1,7 +1,26 @@
...
+
+```python
+py schema_exporter.py [-s SCHEME_NAME] [-p PASSWORD] [-t TNS_NAME] [-o OBJECT_OWNER] [-path PATH_TO_EXPORT] [-tables TABLES]
 ```
+
+| Argument      | Description   |
+| ------------- |-----------------------------------------------|
+| -s            | Denotes oracle scheme for exporting objects   |
+| -p            | Oracle user password in scheme                |
+| -t            | TNSNAME of the Instance                       |
```
Comment (erdemsarigh) [inline README.md:23]: ok

**Review by erdemsarigh — CHANGES_REQUESTED** [review]
not ok

File: README.md (hunk 1 above)
Comment (uurcakmak) [reply README.md:9]: it's ok.

**Comment by uurcakmak** [comment]
done

**Pull request merged**